go/intro/extra-types-ii/README.md

```diff
@@ -0,0 +1,14 @@
+name: More on Types

```

Comment:
Can we make this name a bit more specific too?

Something like: "Using ranges, slices and maps"? I guess the insight "functions are values" is kind of hard to fit into the title.

I'll leave it up to you to decide what to do 👍